ape_vyper/compiler.py

```diff
@@ -92,7 +92,9 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
                     else:
                         raise Exception("No available version to install")
             else:
-                vyper_version = max(self.available_versions)
+                if len(self.installed_versions) == 0:
```

Comment:
`len` check is equivalent to
```suggestion
                if not self.installed_versions:
```